runtime/src/bank.rs

```diff
@@ -5526,9 +5531,21 @@ impl Bank {
             self.last_blockhash().as_ref(),
         ]);
 
-        let epoch_accounts_hash = self.wait_get_epoch_accounts_hash();
-        if let Some(epoch_accounts_hash) = epoch_accounts_hash {
-            hash = hashv(&[hash.as_ref(), epoch_accounts_hash.as_ref().as_ref()]);
+        let accounts_hash_info = if self
```

Comment:
so we always include bank's lt_hash in the bank's hash, unlike epoch_hash, which is only included at one bank in an epoch?